css/css-content/content-animation.html

```diff
@@ -23,29 +27,39 @@
 
 </style>
 <body>
-<div class="target"></div>
 <script>
 
-test(function() {
-  const target = document.querySelector(".target");
-  const style = getComputedStyle(target, "::after");
+const contentAnimationTest = (className, expectedInitialValue, title) => {
```

Comment:
can we not use "initial" again here?

valueBeforeAnimation or valueWithoutAnimation

Reply:
nit: test_content_animation() is more consistent with what WPT typically uses.